spec/controllers/users_controller_spec.rb

```diff
@@ -30,6 +30,20 @@
     end
   end
 
+  describe "#me" do
+    let(:user) { valid_user }
+    context "when it is a valid request", valid_request: true do
+      it "gets single user information" do
+        get :me, id: user.id, format: :json
```

Comment:
I'd presume this route is: `users/me`, not sure why you need the additional `id` param